cli/src/torii/routing.rs

```diff
@@ -389,7 +389,6 @@ async fn handle_metrics(sumeragi: Arc<Sumeragi>, network: Addr<IrohaNetwork>) ->
 async fn handle_status(sumeragi: Arc<Sumeragi>, network: Addr<IrohaNetwork>) -> Result<Json> {
     sumeragi.update_metrics(network);
     let status = Status::from(&sumeragi.wsv_clone().metrics);
-    println!("{:?}", status);
     Ok(reply::json(&status))
 }
 
```

Comment:
> It is probably better to do it after view changes are properly implemented. 

So you have determined that the order of implementation does not matter, right?

Reply:
Yes. I initially thought I that this would depend on having view changes implemented. But all it required was to have block_sync connected to sumeragi again.